hydrolib/core/io/ini/models.py

```diff
@@ -252,6 +252,9 @@ class Comments(INIBasedModel.Comments):
     type: str = Field(alias="type")
     thalweg: Optional[float]
 
+    def _get_identifier(self, data: dict) -> Optional[str]:
+        return data["id"]
```

Comment:
If validation fails because the id is missing, this will raise an additional exception because it cannot find the key.
It is safe to use `data.get("id")`

Reply:
I have fixed all of these in the commit dc72c60 below.